skunkworks_crow/src/main/java/org/odk/share/views/ui/main/MainActivity.java

```diff
@@ -292,4 +294,22 @@ private void setUpLoader() {
             }
         }
     }
+
+    // Add toast "Press BACK to exit"
+    @Override
+    public void onBackPressed() {
+        if(BACK_FLAG){
+            super.onBackPressed();
+            return;
+        }
+        BACK_FLAG = true;
+        Toast.makeText(getApplicationContext(),"Press BACK to exit",Toast.LENGTH_SHORT).show();
```

Comment:
`Press BACK to exit` should be something like `Press BACK again to exit program`, and put the string in `string.xml`.